src/citrine/resources/data_concepts.py

```diff
@@ -611,3 +611,19 @@ def filter_by_name(self, name: str, exact: bool = False,
             params=params,
         )
         return [self.build(content) for content in response["contents"]]
+
+    def delete(self, uid: Union[UUID, str], scope: str = 'id'):
+        """
+        Delete the element of the collection with ID equal to uid.
+
+        Parameters
+        ----------
+        uid: Union[UUID, str]
+            The ID.
+        scope: str
+            The scope of the uid, defaults to Citrine scope ('id')
+
+        """
+        path = self._get_path() + "/{}/{}".format(scope, uid)
+        self.session.delete_resource(path)
+        return True
```

Comment:
What is the value in having this method return something?

Reply:
It has no value; it's just consistent with `src/citrine/resources/project.py`'s way of expressing "I did the thing you wanted"

Reply:
:+1: